examples/ip_sense/main.c

```diff
@@ -7,48 +7,85 @@
 #include <timer.h>
 
 #include <ieee802154.h>
+#include <udp.h>
+
+void print_ipv6(ipv6_addr_t *);
+
+void print_ipv6(ipv6_addr_t *ipv6_addr) {
+  for (int j = 0; j < 14; j += 2)
+    printf("%02x%02x:", ipv6_addr->addr[j], ipv6_addr->addr[j + 1]);
```

Comment:
I'm a little surprised / annoyed that uncrustify didn't fix this automatically, but since I've added some other comments already, could you include `{}` around this loop body